pkgs/by-name/re/rerun/package.nix

```diff
@@ -22,28 +22,32 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "rerun";
-  version = "0.13.0";
-
-  src = fetchFromGitHub {
-    owner = "rerun-io";
-    repo = "rerun";
-    rev = version;
-    hash = "sha256-HgzzuvCpzKgWC8it0PSq62hBjjqpdgYtQQ50SNbr3do=";
+  version = "0.17.0";
+  src = lib.cleanSourceWith {
+    filter = path: type: builtins.baseNameOf path != "Cargo.lock";
+    src = fetchFromGitHub {
+      owner = "rerun-io";
+      repo = "rerun";
+      rev = version;
+      sha256 = "sha256-FrUxGMBau7ak9mK4MYy1LJjmTX4prMHCYYeVXJ69mdg=";
+    };
   };
-  patches = [
-    # Disables a doctest that depends on a nightly feature
-    ./0001-re_space_view_time_series-utils-patch-out-doctests-w.patch
 
+  # patch https://github.com/NixOS/nixpkgs/issues/332957
+  postPatch = ''
+    ln -s ${./Cargo.lock} Cargo.lock
+  '';
 
-    # "Fix cell size test now that the overhead has shrunk"
-    # https://github.com/rerun-io/rerun/pull/5917
-    (fetchpatch {
-      url = "https://github.com/rerun-io/rerun/commit/933fc5cc1f3ee262a78bd4647257295747671152.patch";
-      hash = "sha256-jCeGfzKt0oYqIea+7bA2V/U9VIjhVvfQzLRrYG4jaHY=";
-    })
-  ];
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+  };
 
-  cargoHash = "sha256-qvnkOlcjADV4b+JfFAy9yNaZGaf0ZO7hh9HBg5XmPi0=";
+  cargoBuildFlags = [
+    "--release"
+    "--package rerun-cli"
+    "--no-default-features"
+    "--features native_viewer"
+  ];
```

Comment:
--release is the default for both build and test, so you don't need to specify it.  We have `cargoBuildNoDefaultFeatures`, `cargoBuildFeatures`, and the cargoCheck- equivalents for the feature stuff.